pkg/utils/k8sutil.go

```diff
@@ -87,6 +88,9 @@ func init() {
 
 	ruby24 := runtimeVersion{runtimeID: "ruby", version: "2.4", httpImage: ruby24Http, pubsubImage: ruby24Pubsub}
 	ruby = []runtimeVersion{ruby24}
+
+	netcore2 := runtimeVersion{runtimeID: "netcore", version: "2.0", httpImage: netcore2Http, pubsubImage: ""}
```

Comment:
should we call the runtime "dotnetcore"?

Reply:
Yes, it makes sense. What do you think, @sebgoa and @jjo. Should we change it? If so, should we do this on this PR?

Reply:
done :)